src/Relay.ts

```diff
@@ -29,12 +30,8 @@ class Relay extends EventTarget {
     }
   };
 
-  private createEvent = (message: string) => {
-    return new CustomEvent(message, { detail: {} });
-  };
-
-  public broadcast = (message: MessageObj) => {
-    let event = this.createEvent(message.message);
+  public broadcast = <TPayload = any>(message: MessageObj<TPayload>) => {
+    let event = new CustomEvent(message.message, { detail: message });
 
     if (message?.to) {
       let destination = message.to;
```

Comment:
I think the next line after this (`event.detail["to"] = destination;`) can go away?

Reply:
Good catch!

Reply:
[6482da7](https://github.com/apollographql/apollo-client-devtools/pull/1052/commits/6482da7c9923e794e0b9765fcb60f13387868d91)